STT/corpora.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/corporacreator/corpora.py b/src/corporacreator/corpora.py
+index 6b89946..8e507b6 100644
+--- a/src/corporacreator/corpora.py
++++ b/src/corporacreator/corpora.py
+@@ -40,6 +40,9 @@ class Corpora:
+         """
+         _logger.info("Creating corpora...")
+         corpora_data = self._parse_tsv()
++        # FIXME: THIS IS WRONG DONT USE VALIDATED.TSV AS A SOURCE
```

Comment:
fixme? :)

Reply:
This is your patch ahahha :) I have not touched it.
I'm not even sure why it's there to begin with :')

Reply:
Oh no it was added with
https://github.com/common-voice/commonvoice-fr/commit/f62b21b3d9c9c14723b125ea2331426492fa1d3c